apollo-router/src/http_server_factory.rs

```diff
@@ -70,15 +76,14 @@ impl HttpServerHandle {
         self.server_future.await.map(|_| ())
     }
 
-    pub(crate) async fn restart<Router, PreparedQuery, ServerFactory>(
+    pub(crate) async fn restart<Router, ServerFactory>(
         self,
         factory: &ServerFactory,
-        router: Arc<Router>,
+        router: RouterService<Router>,
```

Comment:
@Geal you missed that one xD should be `service`

Either all the service variables are called service, or they're called router. It sounds minor but consistency helps code readability

Reply:
right, I forgot to update those (I was going to go back and make a kind of trait alias to clean up signatures)